### DISCUSSION TITLE
[3.x]: Is there a way to create list with some criteria for use in custom fields? :: Question or Feature Request

### DISCUSSION THREAD

**Discussion by zzseba78** [craftcms/cms#10966] [Ideas] [answered]
### Description
Hi, i have a case where i need to create relations in entries and categories in Craft, and craft does it very well, but i need to have more control on the elements listed, for example.

Be able to create relationships with list of categories with a specific level in a group - or
List entries in a section that have "featured" ( a custom lightswich field ) activated - or
Or a list of entries that are related to some category.

It´s like create a custom "view" with data, taken from entries or categories and filtering with some criteria, and be able to connect it with a custom field. A concept like Drupal´s "Entity Reference Views".
All this on the CP not the front end.

**Is this possible? is there a plugin to achieve something like this?** Or is something like this on v4?
Thanks.

### Craft CMS version

3.7.38

### Installed plugins and versions

-

**Comment by brandonkelly** [comment, marked as answer]
Are you still wanting to have authors manually select specific entries/categories as relations – just from a more fine-tuned list of available elements to choose from? If so that is now possible in Craft 4 (currently in beta) – see #10393.

Or are you hoping that authors can choose the filters themselves, and the “relationships” are just whatever elements match the author-defined rules?

**Reply by zzseba78** [reply]
Hi @brandonkelly! As a developer, be able to, as you said, "fine tune" list of element that the editors can choose from to create relations. 

I check Relational field conditions, this seems to do exactly what i mean, great work!

**Reply by brandonkelly** [reply]
Sweet!
